server/rpc.go

```diff
@@ -455,7 +454,7 @@ func SubmitSegment(bcast Broadcaster, seg *stream.HLSSegment, nonce uint64) (*ne
 		data = []byte(seg.Name)
 	}
 
-	ti := bcast.GetTranscoderInfo()
+	ti := sess.OrchestratorInfo
```

Comment:
rename `ti` to `oi`?

more generally, can you share the ideas behind using very short variable names? (for example in this case I'm inclined to name this variable at least `oInfo` if not `orchInfo` or even longer :) )

Reply:
> more generally, can you share the ideas behind using very short variable names?

Just a particular style for short-lived local variables. The meaning can be clearly seen from the variable declaration, although you are correct here it would've been better with `oi`.

This is mostly my particular style of programming, but it seems to be shared by the golang community. See https://talks.golang.org/2014/names.slide

Reply:
thanks for sharing your general school of thought! I like the content in the link you shared :)